documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -23,7 +23,7 @@ Container Services for use with Kubernetes* on OCI Compute, and on "Authorized C
 
 [Oracle Linux Cloud Native Environment](https://docs.oracle.com/en/operating-systems/olcne/) is a fully integrated suite for the development and management of cloud-native applications. Based on Open Container Initiative (OCI) and Cloud Native Computing Foundation (CNCF) standards, Oracle Linux Cloud Native Environment delivers a simplified framework for installations, updates, upgrades, and configuration of key features for orchestrating microservices.
 
-WebLogic Server and the WebLogic Kubernetes Operator are certified and supported on Oracle Linux Cloud Native Environment. Operator 2.6.0 provides certified support of OLCNE 1.1 with Kubernetes 1.17.0.
+WebLogic Server and the WebLogic Kubernetes Operator are certified and supported on Oracle Linux Cloud Native Environment. Operator 3.2.5 provides certified support of OLCNE 1.3 with Kubernetes 1.2.0 and CRI-O 1.20.2.
```

Comment:
1.2.0 -> 1.20.2 (Kubernetes and CRI-O versions match).

I'm remembering more of the conversation and I think that we should preserve the statement that operator 2.6.0 is certified on OLCNE 1.1 and that 3.2.5 is certified on OLCNE 1.3.

Reply:
Thank you. Fixed.